content/01.abstract.md

```diff
@@ -1,11 +1,16 @@
 ## Abstract
 
-Professional societies and the conferences that they manage provide an important venue for the dissemination of scientific knowledge.
-Being invited to deliver a keynote at an international society meeting or named a fellow of such a society is a major recognition.
-We sought to understand the extent to which such recognitions reflected the composition of their corresponding field.
-We collected keynote speaker invitations for the international meetings held by the International Society for Computational Biology as well as the names of Fellows, an honorary group within the society.
-We compared these honorees with last and corresponding author contributions in field-specific journals.
-We used multiple methods to estimate the race, ethnicity, gender, and name groupings of authors and the recipients of these honors.
-To address weaknesses in existing approaches, we built a new dataset of more than 700,000 people with name-nationality pairs from Wikipedia and trained long short-term memory neural networks to make predictions.
-Although we observe similar gender distributions between the group of honorees and authors, the proportion of female scientists in the field has not yet reached parity.
-Through each method, we find that white scientists are overrepresented among speakers and honorees, while scientists of color are underrepresented.
+Professional societies and their conferences provide an important venue for disseminating scientific knowledge.
+Delivering a keynote for or being named a fellow by an international society is a major recognition.
+We assess whether such recognitions reflect the composition of the field of bioinformatics.
+We compiled a list of 412 _International Society for Computational Biology_ honorees (75 fellows and 337 keynote speakers).
+We assess the composition of senior researchers by analyzing the names and affiliations of corresponding authors in field-specific journals.
+Comparing to the distribution based on names and affiliations of honorees, we looked for disparities in gender, country of affiliation, race, and name-origin.
+Name-origin predictions were created using a neural network trained on 700,000 name-nationality pairs mined from Wikipedia.
+The proportion of female honorees has kept pace with increasing levels of female authorship, but neither has yet to reach gender parity.
+However, we noticed a striking geographic disparity where the proportion of honorees with an affiliation in the United States was 1.6-fold greater compared to field-specific senior authors.
+In total, we estimate the U.S. received 85 more honorees than would be expected from randomly selecting honorees from senior authors.
+Almost half of the excess in U.S. honorees was accounted for by a deficit of 41 honoree slots from China, France, and Italy.
+Furthermore, within the U.S., we identify racial disparity with an excess of white honorees and a depletion of Asian honorees.
```

Comment:
```suggestion
Furthermore, within the U.S., we identified racial disparity with an excess of white honorees and a depletion of Asian honorees.
```

Reply:
I think this should probably be "disparities" since there are more than one (enrichment of one, depletion of another)